storage/utils/filters.js

```diff
@@ -30,50 +30,56 @@ function filterGenerator(
 	const getValue = filterAlias =>
 		serializer.call(null, null, 'select', filterAlias, fieldName);
 
+	// Allow to use table name as prefix (example: table.field)
+	fieldName = fieldName
+		.split('.')
+		.map(f => `"${f}"`)
+		.join('.');
+
 	switch (filterType) {
 		case filterTypes.BOOLEAN:
-			filters[alias] = `"${fieldName}" = ${getValue(alias)}`;
```

Comment:
Why are you removing `""` from here? I see you're still using different cases on `trs` column names (like "transferData" etc.), and that's the way to validate the case sensitivity.

Reply:
The `""` are being added on the line 36.